templates/scc.yaml.in

```diff
@@ -11,4 +11,4 @@ runAsUser:
 seLinuxContext:
   type: RunAsAny
 users:
-  - system:serviceaccount:{{ .Namespace }}:macvtap-cni
+  - system:serviceaccount:'{{ .Namespace }}':macvtap-cni
```

Comment:
This one looks wrong. We are adding quotes inline to some string we don't own. I doubt OpenShift would be able to eat this. Do we need quotes here?

Reply:
I did this to make the hack/generate.manifests.sh change simple.  It's not the final product, is it? the generated manifests are in manifests/*

Reply:
But these files are generally available to use by anybody as their Go template. So other components (e.g. CNAO) can just consume them and populate. Replacing quotes should not a be a part of the workflow. Can we make it so templates are still valid (replacing variables in Go is enough)?

Reply:
I see, falling back to only adding the single quotes to the macvtap.yaml.in and adding removing them in generate-manifests.sh